harpy/_validations.py

```
@@ -300,82 +301,92 @@
+                        end = int(row[2])
+                    except:
+                        print_error("invalid format", f"The input file is formatted incorrectly at line {idx}. This is the first row triggering this error, but it may not be the only one.")
+                        print_solution_with_culprits(
+                            f"Rows in [blue]{regioninput}[/] need to be [bold]space[/] or [bold]tab[/] delimited with the format [yellow bold]contig start end[/] where [yellow bold]start[/] and [yellow bold]end[/] are integers.",
+                            "Rows triggering this error:"
+                            )
+                        click.echo(line, file = sys.stderr)
+                        sys.exit(1)
+                contigs = contig_lens(genome)
+                if row[0] not in contigs:
+                    print_error("missing contig", f"The contig listed at row {idx} ([bold yellow]{row[0]}[/]) is not present in ([blue]{os.path.basename(vcf)}[/]). This is the first row triggering this error, but it may not be the only one.")
```

Ruff (0.8.2)

Undefined name `vcf`

(F821)